index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles/reset.css">
+  <link rel="stylesheet" href="styles/base.css">
+  <link rel="stylesheet" href="styles/layout.css">
+  <link rel="stylesheet" href="styles/modules.css">
+</head>
+
+<body>
+    <header>
+        <div id="a">a</div>
+    </header>
+
+    <main>
+        <div id="b">b</div>
+        <div id="c">c</div>
+        <div id="d">d</div>
+        <div id="e">e</div>
+        <div id="f">f</div>
+        <div id="g">g</div>
+        <div id="h">h</div>
+    </main>
+
+    <footer>
+        <div id="i">i</div>
+    </footer>
+</body>
```

Comment:
The red symbols mean that there is no empty line at the bottom of the file. Leave an empty line at the end of files is a good habit to get into.